server/src/types/bots.js

```diff
@@ -15,6 +15,11 @@ const bot = new graphql.GraphQLObjectType({
     agentId: { type: graphql.GraphQLString },
     userId: { type: graphql.GraphQLString },
 
+    jwt: {
+      type: graphql.GraphQLNonNull(graphql.GraphQLString),
+      resolve: require('../resolvers/authentications').getJwtByAgentId,
+    },
```

Comment:
It seems like the jwt should only be available to the owner. I'm not sure how to do this in permissions, but it seems important to figure out. (row-level permissions)

Reply:
Here we can use low-level restrictions (for instance users can see only its bots) or use "shield" permissions.

Reply:
I think users should be able to see all bots (the names/Ids/descriptions), but not the jwt token. Any idea on the best way of doing that? Not sure if shield helps with that.

Reply:
Fixed.

Reply:
Fixed.